### PR TITLE
Update DIP-3.md

### PR DESCRIPTION
**Pull request by BettinaBF**
Thanks for this DIP and for being part of the first teams to try out the DIPs! The general feedback from the Devcon team is that we really enjoy the concept and possibilities your DIP offers! Our team did, however, have some concerns, as described by a few select comments from our team pasted word-for-word below:

- I can see the good idea behind it and cutting out the middle man, but travel booking is one of the things where you need REALLY good UX and also customer service etc in case of questions or errors. I'm not sure [how they plan] to provide this kind of standard. ... Also, I would need to see this working irl to be convinced. On the glider website I wasn't even able to click on the T&Cs.
- The glider website doesn't seem very functional to me, it has basically only one service provider.
- [Redacted] makes a good point re: buying ticket & hotel/flight together. Customer service is a concern as well. Built-in search for discounted hotel & flight rates is a cool idea, but we'd need a very smooth UX.

In summary, we like the *goals* of the DIP and would like to help make this use case possible for Devcon attendees. We'll happily connect you with our hotel contacts in Bogota, and provide support where possible to make this idea a reality. But at this time, we cannot promise specific inclusion of this on the [devcon.org](http://devcon.org) website. That will be contingent on us seeing a great user experience built and a plan for how to handle customer service. We'll wait to see a more fleshed out implementation of this application, and in the meantime we will start connecting you with our hotel contacts!

In practical termes, please update your DIP operational requirements to Devcon.org implementation being contingent on your future developments and we'll be set to validate it! 

Thanks

### CODE REVIEW

**Comment by kvakes** [comment]
Hi @BettinaBF!

Thank you for your feedback, we appreciate your trust in our team.

## Stellar UX

You are 100% correct. While Glider is just a POC (with limited UX and content), we have a dedicated UX designer who will be working with your website team, as well as dedicated developer implementing the required features.

## Customer Support (part of the UX)

We will provide the following:

- Q&A / Support documentation on the website
- Live Chat support
- Email support
- Phone support

At peak times (around conference) we will have several customer support agents available.

## Hotel Connections

Our team is already in contact with some of the hotels. Thank you for those connections!

**Review by kvakes — APPROVED** [review]
